src/rpc/util.cpp

```diff
@@ -113,6 +113,23 @@ std::vector<unsigned char> ParseHexO(const UniValue& o, std::string strKey)
     return ParseHexV(find_value(o, strKey), strKey);
 }
 
+CoinStatsHashType ParseHashType(const UniValue& param, const CoinStatsHashType default_type)
+{
+    if (param.isNull()) {
+        return default_type;
+    } else {
```

Comment:
style-nit: No need for the `else` and all the whitespace padding below

```suggestion
    }
    const std::str...
```

Reply:
Thanks, now fixed in #19145.